src/components/BookList.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { v4 as uid } from 'uuid';
+import styles from '../css/bookList.module.css';
+import Book from './Book';
+
+const BookList = () => {
+  const bookList = [
+    {
+      Id: uid(),
+      Type: 'Action',
+      Name: 'The Hunger Game',
+      Author: 'Suzanne Collins',
+      Chapter: 'Chapter 8',
+      Progress: '64%',
+    },
+
+    {
+      Id: uid(),
+      Type: 'Science Fiction',
+      Name: 'Dune',
+      Author: 'Frank Herbert',
+      Chapter: 'Chapter 17',
+      Progress: '64%',
+    },
+
+    {
+      Id: uid(),
+      Type: 'Economy',
+      Name: 'Capital in the Twenty-First Century',
+      Author: 'Rozer Fedric',
+      Chapter: 'Chapter 11',
+      Progress: '64%',
+    },
+
+  ];
+  return (
+    <ul className={styles.bookContainer}>
+      {bookList.map((book) => (
+        <li key={book.Id}>
+          <Book
+            name={book.Name}
+            type={book.Type}
+            author={book.Author}
+            chapter={book.Chapter}
+            progress={book.Progress}
+          />
+        </li>
+      ))}
+    </ul>
```

Comment:
- [ ] @ibugithub great work on your pages, however, you forgot to add a component that should be imported to booklist components, the component is supposed to have a form to create new books.
- [ ] @ibugithub I also have a logic directory containing files used in the math-magician project kindly consider deleting them, to ensure your project contains only relevant information, also remove the Auth page since it's not part of the requirements, and its done wrongly.

Reply:
The 'Add Book' form has been added.
Deleted the logic folder and Auth page.